### DISCUSSION TITLE
Add Magenta Line to Minimap

### DISCUSSION THREAD

**Discussion by KBNOtaku** [magknight/787aviatorsTracker#776] [Ideas]
**Is your feature request related to a problem? Please describe.**
No Magenta Route line is visible on the Minimap, only way points.

**Describe the solution you'd like**
The Magenta Route line should be visible on the Minimap.

**Additional context**
 - OS:  Windows 10 ]
 - X-Plane version [ 11.55r2 ]
 - Version [ 1.7.18-alpha6 ]

**Comment by aeroniemi** [comment]
this is kinda intentional - we can't add the line there as it'd require limiting the range of all other NDs to 20nm, which isn't really a good idea
